packages/itinerary-body/src/ItineraryBody/index.js

```diff
@@ -49,6 +49,9 @@ const ItineraryBody = ({
       // Create a row containing this leg's start place and leg traversal details
       rows.push(
         <PlaceRow
+          AlertToggleIcon={AlertToggleIcon}
+          AlertBodyIcon={AlertBodyIcon}
+          accessibilityScoreGradationMap={accessibilityScoreGradationMap}
           // eslint-disable-next-line react/no-array-index-key
           key={i + (isDestination ? 1 : 0)}
           config={config}
```

Comment:
Sort props

Reply:
Sorted in d8db56ebd3f28f69acd09554ca296d72ff3a3ee6